docs: Privategpt integration docs. (#412)
qdrant/landing_page: qdrant-landing/content/documentation/integrations/privategpt.md

---
title: PrivateGPT
weight: 1600
---

# PrivateGPT

> [PrivateGPT](https://docs.privategpt.dev/) is a production-ready AI project that allows you to ask questions about your documents using the power of Large Language Models (LLMs), even in scenarios without an Internet connection. 100% private, no data leaves your execution environment at any point.

Author: Should be resolved now.
https://github.com/qdrant/landing_page/pull/412/commits/eaac4143bec6f9d4c72dd7178491690d44bddd36.

PrivateGPT supports Qdrant as a vectorstore for ingesting and retrieving documents.

## Usage

> To enable Qdrant, set the vectorstore.database property in the settings.yaml file to qdrant and install the qdrant extra within your PrivateGPT project.

Reviewer: ```suggestion
To enable Qdrant, install the qdrant extra within your PrivateGPT project first.
```
